scripts/system_performance/models/query_argument_constructor.py

```diff
@@ -0,0 +1,116 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Contains query constructors for system performance check MySQL queries.
+"""
+
+# Dependencies
+from datetime import date, datetime
+from calendar import monthrange
+
+# from scripts.system_performance.data_persistence.reduction_run_queries_test import DatabaseMonitorChecks
+from scripts.system_performance.data_persistence.system_performance_queries import DatabaseMonitorChecks
+
+
+def get_day_of_week():
+    # print(date.today())
+    # todays_date = '2020-02-14'
+    # temp = date(*map(int, todays_date.split('-')))
+    return date.today()
+
+# def get_last_day_of_month():
+#
+#     pass
+
+
+def get_list_of_instruments():
```

Comment:
could be fine as `get_instruments():` the import path suggests that it will be from the database and thus we can assume a list